packages/synapse-react-client/vite.config.ts

```diff
@@ -1,48 +1,25 @@
-import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfill'
-import react from '@vitejs/plugin-react'
 import { resolve } from 'path'
-import { defineConfig } from 'vite'
-import svgr from 'vite-plugin-svgr'
-import jotaiDebugLabel from 'jotai/babel/plugin-debug-label'
-import jotaiReactRefresh from 'jotai/babel/plugin-react-refresh'
+import { mergeConfig } from 'vite'
+import viteConfig from 'vite-config'
+import { externalizeDeps } from 'vite-plugin-externalize-deps'
+import dts from 'vite-plugin-dts'
 
-export default defineConfig({
-  root: './src',
+export default mergeConfig(viteConfig, {
```

Comment:
Creates the ESM and CJS bundles and the TypeScript module definition file using Vite